Add more POIs in Canada

andrewpmk opened the pull request:
Added POIs:
- Access Storage
- Active Green + Ross
- Ali Baba's
- Becker's
- Big Bear/Big Bee
- Bikini Village
- BioPed
- Blinds to Go
- Bouclair Home
- Bourbon St. Grill
- BrokerLink
- Burrito Boyz
- Maison Birks
- Mister Transmission
- Mobile Klinik
- Moxie's
- Quickie
- Sirens
- South Street Burger
- Stinson
- Sun Life Financial
- The Works
- Trade Secrets
Updated POIs:
- 9Round
- Allstate
- Alterna Savings
- Boathouse
- Nails For You

AntiCompositeNumber commented on data/brands/shop/window_blind.json:
```diff
@@ -4,6 +4,17 @@
+    {
+      "displayName": "Blinds to Go",
+      "id": "blindstogo-a6607a",
+      "locationSet": {"include": ["ca"]},
```
```suggestion
      "locationSet": {"include": ["ca-on.geojson", "us-ct.geojson", "us-de.geojson", "us-ma.geojson", "us-md.geojson", "us-nj.geojson", "us-ny.geojson", "us-pa.geojson", "us-va.geojson"]},
```
per https://www.blindstogo.com/en/stores

AntiCompositeNumber replied:
Please also add an entry for Quebec per https://www.blindstogo.com/fr/magasins

AntiCompositeNumber reviewed (changes requested):
In the future, smaller, more focused pull requests are easier to review.

Cj-Malone commented on data/brands/shop/interior_decoration.json:
```diff
@@ -17,6 +17,17 @@
         "shop": "interior_decoration"
       }
     },
+    {
+      "displayName": "Bouclair Home",
+      "id": "bouclairhome-aa1ae6",
+      "locationSet": {"include": ["ca"]},
+      "tags": {
+        "brand": "Bouclair",
+        "brand:wikidata": "Q20638837",
+        "name": "Bouclair Home",
+        "shop": "interior_decoration"
+      }
+    },
```
In theory `name` should be the [facia](https://www.google.com/maps/place/Bouclair+Home/@43.841389,-79.4211637,3a,75y,350.68h,93.34t/data=!3m7!1e1!3m5!1sTsTvVsXs7ve9J7Ix9ZEj2Q!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3DTsTvVsXs7ve9J7Ix9ZEj2Q%26cb_client%3Dmaps_sv.tactile.gps%26w%3D203%26h%3D100%26yaw%3D341.89697%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192!4m10!1m2!2m1!1sBouclair+Home!3m6!1s0x882b2b836f0272ff:0x1bbb23ec256ca6e4!8m2!3d43.8415112!4d-79.4212305!15sCg1Cb3VjbGFpciBIb21lIgOIAQFaDyINYm91Y2xhaXIgaG9tZZIBD2Z1cm5pdHVyZV9zdG9yZeABAA!16s%2Fg%2F1tjl6jt0?entry=ttu) and `brand` the ["brand"](https://www.bouclair.com/en/).

However OSM and NSI aren't prefect in this.

andrewpmk replied:
They're different brands of store operated by the same company. NSI keeps asking me to change "Bouclair Home" stores to "Bouclair" which is incorrect.
We have the same issue with "The Brick" vs "The Brick Mattress Store".

andrewpmk replied:
Added French entry

andrewpmk commented on data/brands/shop/convenience.json:
```diff
@@ -617,6 +628,29 @@
+      },
+      "tags": {
+        "brand": "Big Bear",
+        "brand:wikidata": "Q123409887",
```
Looks like they are separate chains https://www.reddit.com/r/Hamilton/comments/4bw8rt/big_bear_vs_big_bee_stores/
They're chains of convenience stores near Hamilton Ontario (also Niagara Falls, St Catharines, Kitchener etc.)

andrewpmk replied:
Removed Big Bear. Looks like it is not the same chain https://www.reddit.com/r/Hamilton/comments/4bw8rt/big_bear_vs_big_bee_stores/

andrewpmk commented on data/brands/leisure/fitness_centre.json:
```diff
@@ -61,8 +61,39 @@
-      "locationSet": {"include": ["us"]},
+      "id": "9round-23f8d6",
+      "locationSet": {
+        "include": [
```
Changed to 001

andrewpmk commented:
Updated pull request with suggested changes.
Removed Big Bear because it appears to be different from Big Bee.
Added French entries for Bouclair Home and Blinds to Go.

AntiCompositeNumber commented on data/brands/shop/window_blind.json:
```diff
@@ -52,6 +64,20 @@
+      "matchNames": ["blinds to go"],
+      "tags": {
+        "brand": "Le Marché du Store",
+        "brand:wikidata": "Q113686682",
```
```suggestion
        "brand:wikidata": "Q123409913",
```

andrewpmk commented:
Updated brand:wikidata of Blinds to Go to Q123409913
